src/template.html

```diff
@@ -42,9 +42,25 @@ <h3 class="movies-length"></h3>
     </header>
     <main>
       <div class="main-content">
-       
-        
+              
       </div>
+      <!-- <div class="modalBox">
+         <div class="modalContentWrapper">
+           <button class="modal-close-btn">x</button>
+           <div class="image-box">
+            <p>No image found</p>
+           </div>
+           <p class="pop-title">Movie title</p>
+           <div class="info">
+             <p>Likes: 453</p>
+             <p>Type: Romans, Love</p>
+             <p>Language: english</p>
+             <p>Official Site: visit site</p>
+           </div>
+           <p class="comments-head">Comments</p>
+           <hr>
+         </div>
+      </div> -->
```

Comment:
Hi Victor, Kindly remove the commented-out code as it makes the code looks unclean.

Reply:
Great! I will remove it in my next commit ASAP.